types/koa-better-body/index.d.ts

```diff
@@ -25,6 +26,11 @@ declare namespace KoaBetterBody {
     decodeURIComponent?: any;
     maxKeys?: number;
     buffer?: boolean;
+    QueryString?: string;
+    encoding?: string;
```

Comment:
I'm not seeing `encoding` as an option anywhere in the [defaultOptions](https://github.com/tunnckoCoreLabs/koa-better-body/blob/e5f6f677a93b6acb39d1173dea3d5cf2f917c035/utils.js#L20). Perhaps you are trying to write definitions for *[this](https://github.com/guileen/koa-better-body)* `koa-better-body`. :thinking:

Reply:
Sorry my poor English. <[Here](https://github.com/tunnckoCoreLabs/koa-better-body/blob/master/recipes/multipart/README.md)> is a reference of the options.🤔

`encoding`, `uploadDir `,  `keepExtensions` are using as options in _multipart_